### PR TITLE
remove unneed parameter in test struct

### PR DESCRIPTION
**Pull request by stewart-yu**
**What this PR does / why we need it**:
remove unneed parameter in test struct

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubeapiserver/options/storage_versions_test.go [hunk 1]
```diff
@@ -76,3 +71,36 @@ func TestGenerateStorageVersionMap(t *testing.T) {
...
+				{Version: "v1"},
+				{Group: "extensions", Version: "v1beta1"},
+			},
+			expectedPreferredVersionsString: "extensions/v1beta1,v1",
+		},
+		{
+			groupVersion: []schema.GroupVersion{
+				{Version: "v1"},
+				{Group: "extensions", Version: "v1beta1"},
+				{Group: "extensions", Version: "v1beta1"},
```
Comment (caesarxuchao) [inline pkg/kubeapiserver/options/storage_versions_test.go]: This isn't a valid case.

File: pkg/kubeapiserver/options/storage_versions_test.go [hunk 2]
```diff
@@ -76,3 +71,36 @@ func TestGenerateStorageVersionMap(t *testing.T) {
 		}
 	}
 }
+
+func TestToPreferredVersionString(t *testing.T) {
```
Comment (caesarxuchao) [inline pkg/kubeapiserver/options/storage_versions_test.go]: The function is very simple. A unit test doesn't seem necessary. WDYT?

Comment (stewart-yu) [reply pkg/kubeapiserver/options/storage_versions_test.go]: MapToString seems test well in others place, remove seems OK

**Pull request merged**